core/main/src/Core/Particle.ts

```diff
@@ -159,7 +159,9 @@ export class Particle implements IParticle {
         this.fill = this.shapeData?.fill ?? this.fill;
         this.close = this.shapeData?.close ?? this.close;
         this.particlesOptions = particlesOptions;
-        this.zIndexFactor = (this.particlesOptions.zIndex + 100) / 100;
+        this.zIndexFactor = this.particlesOptions.zIndex.value;
```

Comment:
Assign directly the ClampedZIndexFactor here, it's a factor value so it will be used in arithmetical operations

Reply:
👍